src/test/kotlin/no/nav/syfo/application/service/VarselServiceSpek.kt

```diff
@@ -0,0 +1,92 @@
+package no.nav.syfo.application.service
+
+import io.mockk.*
+import no.nav.syfo.ExternalMockEnvironment
+import no.nav.syfo.UserConstants
+import no.nav.syfo.generator.generateVarsel
+import no.nav.syfo.infrastructure.database.VarselRepository
+import no.nav.syfo.infrastructure.database.dropData
+import no.nav.syfo.infrastructure.kafka.esyfovarsel.ArbeidstakervarselProducer
+import org.amshove.kluent.shouldBeEqualTo
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.RecordMetadata
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.util.concurrent.Future
+
+private const val journalpostId = "123"
+private val unpublishedVarsel = generateVarsel().copy(journalpostId = journalpostId)
+
+class VarselServiceSpek : Spek({
```

Comment:
Kan erstatte mockingen av repository her senere når vi har kode for å sette varsler journalført på plass (da kan man først opprette disse i databasen og bruke de i testen)